### PR TITLE
Fix changelog fragment inconsistencies + add deprecation changelog fragment for 924

### PR DESCRIPTION
**Pull request by davfsa**

### CODE REVIEW

File: changes/984.feature.md [hunk 1]
```diff
@@ -1 +1 @@
-old_x keyword arguments in the event factory now default to None.
+`old_x` keyword arguments in the event factory now default to `None`.
```
Comment (FasterSpeeding) [inline changes/984.feature.md:1]: You know this doesn't refer to anything in code right

File: changes/955.feature.md [hunk 2]
```diff
@@ -1 +1 @@
-Added support for GET /users/@me/guilds/{guild}/member
+Added support for GET `/users/@me/guilds/{guild}/member`.
```
Comment (FasterSpeeding) [inline changes/955.feature.md:1]: Shouldn't get also be in the code blocks also why put it in code blocks if it's not referring to anything in code

File: changes/924.deprecation.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Deprecated `RESTClient.command_builder` and `RESTClient.create_application_command`.
+
```
Comment (FasterSpeeding) [inline changes/924.deprecation.md:2]: The clear menu methods have nothing to do with this deprecation

**Comment by FasterSpeeding** [comment]
There's no style guide for these entries so how can there be anything to be "consistent" with

File: changes/924.deprecation.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
...
+Instead, you are encouraged to switch to `RESTClient.slash_command_builder`/`RESTClient.context_menu_builder` and `RESTClient.create_slash_command`/`RESTClient.create_context_menu_command` respectively.
```
Comment (FasterSpeeding) [inline changes/924.deprecation.md:3]: Encouraged is the wrong word here, you should switch its a deprecation not a "this new method is preferred"

File: changes/924.feature.md [hunk 5]
```diff
@@ -1 +1 @@
-Add new command types and application autocomplete option
+Add new command types and application autocomplete option.
```
Comment (thesadru) [inline changes/924.feature.md:1]: wouldn't something like "Add context menu commands and command autocomplete." be better? Who even wrote this?

File: changes/924.deprecation.md (hunk 3 above)
Comment (davfsa) [reply changes/924.deprecation.md:2]: What exactly do you mean?

File: changes/955.feature.md (hunk 2 above)
Comment (davfsa) [reply changes/955.feature.md:1]: Its just for formatting, but ig it can be left how it was

File: changes/984.feature.md (hunk 1 above)
Comment (davfsa) [reply changes/984.feature.md:1]: Again, formatting, but will revert

File: changes/924.deprecation.md (hunk 3 above)
Comment (FasterSpeeding) [reply changes/924.deprecation.md:2]: command_builder is a deprecated alias for slash_command_builder and create_application_command is a deprecated alias for create_slash_command; create_context_menu_command and context_menu_builder are unrelated new functionality

**Review by FasterSpeeding — APPROVED** [review]

**Pull request merged**